index.bs

```diff
@@ -1050,6 +1052,7 @@ Possible values are:
 <ul dfn-for="source debug data type">
 <li>"<dfn><code>source-destination-limit</code></dfn>"
 <li>"<dfn><code>source-destination-rate-limit</code></dfn>"
+<li>"<dfn><code>source-invalid-randomized-response-config</code></dfn>"
```

Comment:
I'd prefer naming that is less tied to the `randomized response` terminology. `source-output-space-limit`, or something consistent with terminology we use in explainers or other docs? Not sure, but I think RR is too specific.

@johnivdel may have ideas here.

Reply:
Updated to `source-output-space-limit`, thanks.